### PR TITLE
Fix deprecation warning for Rails 4.2.

### PR DESCRIPTION
**Pull request by JuanitoFatas**
Related commit on rails/rails:

https://github.com/rails/rails/commit/d35f0033c7dec2b8d8b52058fb8db495d49596f7

- Update travis.yml.
- Add 4.2 Gemfile

### CODE REVIEW

File: lib/unread/readable.rb [hunk 1]
```diff
@@ -112,7 +112,7 @@ def unread?(user)
             true
           end
         else
-          !!self.class.unread_by(user).exists?(self) # Rails4 does not return true/false, but nil/count instead.
+          self.class.unread_by(user).exists?(self.id)
```
Comment (JuanitoFatas) [inline lib/unread/readable.rb]: Rails 4.2 `exists?` will return true now.

**Comment by jrmsklar** [comment]
Thanks for this fix @JuanitoFatas! @ledermann do you have a timeline for when a new release will be made with this addition? 

Thanks

**Comment by ledermann** [comment]
Just released v0.5.0

**Comment by jrmsklar** [comment]
Great, thanks @ledermann.

**Comment by JuanitoFatas** [comment]
Thanks! :+1:

**Pull request merged**